src/highlight_canvas.class.js

```diff
@@ -88,6 +88,17 @@
       return this.insertAt(path, highlightLayerIndex);
     },
 
+    /**
+     * Adds background to the stage right after the drawing highlight layer
+     * @param {Object} object Object to insert
+     * @return {Self} thisArg
+     * @chainable
+     */
+    addBackground: function(background) {
+      var backgroundLayerIndex = this.getHighlightLayerIndex() + 1;
```

Comment:
I like how the not-found case (which shouldn't happen) returns -1 and +1 goes to 0 and would result in adding the BG at the start (which seems good).